tests/utils.test.ts

```diff
@@ -41,12 +47,11 @@ describe('bin/utils', () => {
         ],
       });
       expect(tableOutput).toBe(
-        'value1\tvalue2\ndata1 \tdata2\nundefined\tundefined\n',
+        'value1\tvalue2\n' + 'data1 \tdata2\n' + 'N/A   \tN/A\n',
```

Comment:
Why concat the string here, can't it be all one string?

Reply:
it's been made into just one string